locations/scharni.yml

```diff
@@ -1,4 +1,52 @@
 ---
+location: scharni
+location_nice: Scharni
+latitude: 52.512702825706896
+longitude: 13.464007297614232
+community: true
+
+hosts:
+  - hostname: scharni-core
+    role: corerouter
+    model: "zyxel_nbg6617"
+
+  - hostname: scharni-ap-gastwg
+    role: ap
+    model: "ubnt_unifi-6-lite"
+
+  - hostname: scharni-ap-keller
+    role: ap
+    model: "tplink_eap225-outdoor-v1"
+
+  - hostname: scharni-ap-kneipe
+    role: ap
+    model: "zyxel_nwa50ax"
+
+  - hostname: scharni-ap1
+    role: ap
+    model: "tplink_archer-c6-v2"
+
+  - hostname: scharni-ap2
+    role: ap
+    model: "tplink_archer-c6-v2"
+
+  - hostname: scharni-ap3
+    role: ap
+    model: "mikrotik_sxtsq-5-ac"
+    mac_override:
+      eth0: 48:8f:5a:b9:83:82
+
+
+snmp_devices:
+  - hostname: scharni-sama
+    address: 10.31.52.163
+    snmp_profile: airos_8
+
+  - hostname: scharni-zwingli
+    address: 10.31.52.164
+    snmp_profile: airos_6
```

Comment:
this is an airos_8 device. please update this.

Reply:
Wollen wir nicht lieber mal separat die Locations durchgehen, anstatt bei jedem Singlefile-PR zusätzliche Arbeit draufzuladen? Ist doch nervig wenn die Singlefile-PRs länger rumliegen als notwendig.